binary_to_decimal/binary_to_decimal.js

```diff
@@ -0,0 +1,53 @@
+// ----------------------------------
+// Function binaryToDecimal
+
+function binaryToDecimal(binary_number){
+    const length = binary_number.toString().length; 
+    const string = binary_number.toString();
+    let binaryVal = true; //default value
+
+    for(let i = 0; i < length; i++){  //binary check - compare string
```

Comment:
Tenhle C-like zápis cyklu je samozřejmě v pořádku, ale JS nabízí modernější variantu `for (let i of string)`. Máš přímo přístup k iterované proměnné skrz `i` a nemusíš jí indexovat takto `string[i]`.

Tady si o tom můžeš přečíst více: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of

Reply:
Tieto for loopy mi doteraz prišli neintuitívne, preto som používal 'C-like zápis', preto že to bolo jednoduché. Keď je človek lenivý, nechce sa mu to ináč, ale pritom je pravda že existujú jednoduchšie spôsoby, ako to urobiť.